docs/.vuepress/components/WrapLayoutLiveDemoDoc.vue

```diff
@@ -0,0 +1,346 @@
+          <TextField
+            v-model="label.text"
+            width="80"
+            :id="'wraplayout_childreneditor_text_input' + index"
+          ></TextField>
+
+          <Label text="Width: " class="m-g-3"></Label>
+          <TextField
+            keyboardType="number"
+            v-model="label.width"
+            width="80"
+            :id="'wraplayout_childreneditor_width_input' + index"
+          ></TextField>
+
+          <Label text="Height: " class="m-g-3"></Label>
+          <TextField
+            keyboardType="number"
+            v-model="label.height"
+            width="80"
+            :id="'wraplayout_childreneditor_height_input' + index"
+          ></TextField>
+
+          <Label text="Color: " class="m-g-3"></Label>
+          <TextField
+            type="color"      
+            v-model="label.bgColor"
+            width="100"
+            :id="'wraplayout_childreneditor_bgColor_input' + index"
+          />
+
+          <Button
+            class="fa fa-remove"             
+            @tap="removeItem(index)"
+            :id="'wraplayout_childreneditor_removeItem_button' + index"
+          ></Button>
+        </StackLayout>
+      </StackLayout>
+    </StackLayout>
+
+    <StackLayout class="demo-panel">
+      <Label class="demo-panel__title" text="Rendered Result"></Label>
+      <WrapLayout
+        :width="width"
+        :height="height"
+        :itemWidth="containerItemWidth"
+        :itemHeight="containerItemHeight"
+        :orientation="orientation"
+        :backgroundColor="containerBackground"
+        id="wraplayout_thewraplayout"
+        class="wrap_demo_item"
+      >
+        <Label
+          v-for="(label, index) in labels"
+          :id="'dynamic-generated-label' + ' label_'+ index"
+          :text="label.text"
+          :key="label.text + '__' + index"
+          :width="label.width"
+          :height="label.height"
+          :backgroundColor="label.bgColor"
+        ></Label>
+      </WrapLayout>
+    </StackLayout>
+  </StackLayout>
+</template>
+
+
+<script>
+import Label from '../../../src/components/Label';
+import Button from '../../../src/components/Button';
+import TextField from '../../../src/components/TextField';
+import WrapLayout from '../../../src/layouts/WrapLayout';
+import StackLayout from '../../../src/layouts/StackLayout';
+
+class ColorAlternator {
+  currentIndex = 0;
+  items = ['#43b883', '#1c6b48', '#289062', '#49e95e', '#33745e'];
+  constructor(colors) {
+    if (colors) {
+      this.items = colors;
+    }
+  }
+  next() {
+    if (this.currentIndex < 0) {
+      this.currentIndex = 0;
+    }
+    if (this.items.length < this.currentIndex + 1) {
+      this.currentIndex = 0;
+    }
+    return this.items[this.currentIndex++];
+  }
+
+  back() {
+    this.currentIndex--;
+  }
+}
+
+const colorAlternator = new ColorAlternator();
+
+export default {
+  name: 'WrapLayoutLiveDemoDoc',
+  data() {
+    return {
+      labels: [],
+      width: 660,
```

Comment:
can it be problem for mobile devices ? since it may be bigger than screen width, It can overflow the screen ?

Reply:
lowered the initial width to the 460px

Reply:
resolved test